UniFi Device Blocker/module.php

```diff
@@ -148,9 +148,13 @@ public function AuthenticateAndProcessRequest() {
 					}
 					else if ($code == 400) {
 						$this->SendDebug($this->Translate("Authentication"),$this->Translate('400 Bad Request - The server cannot or will not process the request due to an apparent client error.'),0);
+						echo $this->Translate('400 Bad Request - The server cannot or will not process the request due to an apparent client error.');
+						exit();
```

Comment:
@elueckel 
Da wir uns in einer Funktion befinden, wäre folgendes zu bevorzugen:
`return false;`

Und am Ende im Idealfall noch für die erfolgreiche Ausführung:
`return true;`

Reply:
> @elueckel Da wir uns in einer Funktion befinden, wäre folgendes zu bevorzugen: `return false;`
> 
> Und am Ende im Idealfall noch für die erfolgreiche Ausführung: `return true;`

change done